narwhal/worker/src/client.rs

```diff
@@ -0,0 +1,67 @@
+use std::sync::Arc;
+
+use arc_swap::ArcSwapOption;
+use thiserror::Error;
+use types::{metered_channel::Sender, Transaction, TxResponse};
+
+static LOCAL_NARWHAL_CLIENT: ArcSwapOption<LocalNarwhalClient> = ArcSwapOption::const_empty();
```

Comment:
Yes I'm adding a map to differentiate between different instances of Narwhal. Btw with multiple workers, is the plan to share a single TxServer?

Reply:
I don't think so, as I said on my review comment especially I believe for the local mode we shouldn't even bootstrap the TxServer (the gRPC one).  Now, when we'll have multiple local workers we can load balance between them via some logic on the ConsensusAdapter? (or some other component).

Reply:
I think adding the logic to disable TxServer may not have much benefits, since it does not cost much CPU and we may have both local and remote clients to Narwhal in future.

Reply:
I'd be worried that if we disable tx server then it will bit rot. I think we should either have it run (even if not in use) or else delete the code.